### PR TITLE
LSF docs

### PR DESCRIPTION
**Pull request by al-rigazzi**
This PR adds documentation for LSF
- `JsrunSettings`, `BsubBatchSettings`
- `LSFOrchestrator`
- `LSFLauncher`

### CODE REVIEW

File: doc/launchers.rst [hunk 1]
```diff
@@ -241,3 +243,34 @@ As well as batch settings for ``qsub`` through:
...
+
+---------------------------------------------------------------------
+
+LSF
+===
+
+The LSF Launcher works like the PBSPro and Cobalt launchers and
+is compatible with LSF and OpenMPI workloads.
+
+To use the LSF launcher, specify at ``Experiment`` intialization:
```
Comment (EricGustin) [inline doc/launchers.rst:255]: initialization

File: doc/overview.rst [hunk 2]
```diff
@@ -61,7 +61,7 @@ The key features of the IL are:
   - An API to start, monitor, and stop HPC jobs from Python or Jupyter notebook.
   - Automated deployment of in-memory data staging (Redis) and computational
     storage (RedisAI)
-  - Programmatic launches of batch and in-allocation jobs on PBS, Slurm,
+  - Programmatic launches of batch and in-allocation jobs on PBS, Slurm, lSF,
```
Comment (EricGustin) [inline doc/overview.rst:64]: `lSF` -> `LSF`

Comment (al-rigazzi) [reply doc/overview.rst]: Thanks

File: doc/launchers.rst (hunk 1 above)
Comment (al-rigazzi) [reply doc/launchers.rst]: Thanks

File: doc/api/smartsim_api.rst [hunk 3]
```diff
@@ -222,6 +224,70 @@ that of the ``QsubBatchSettings`` for PBSPro.
...
 
 ------------------------------------------
+    
+    
+JsrunSettings
+-------------
+
+.. _jsrun_api:
+
+``JsrunSettings`` can be used on any system that suppports the
```
Comment (EricGustin) [inline doc/api/smartsim_api.rst:234]:
```suggestion
``JsrunSettings`` can be used on any system that supports the
```

**Comment by Spartee** [comment]
Closing as this has been merged into #71 for convenience.

**Pull request closed without merging**